tools/dotcms-cli/cli/src/test/java/com/dotcms/config/EnvironmentConfigSource.java

```diff
@@ -0,0 +1,48 @@
+package com.dotcms.config;
+
+import org.eclipse.microprofile.config.spi.ConfigSource;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+import java.util.Set;
+
+public class EnvironmentConfigSource implements ConfigSource {
```

Comment:
doc

Reply:
Probably PropertiesConfigSource is a better name

Reply:
Perfect, I'll change it.